docs/lib/storage/fragments/js/copy.md

```diff
@@ -0,0 +1,116 @@
+## Copy
+
+`Storage.copy` method copies an existing file to a different key.
+
+It returns a `{key: new S3 Object key}` object on success:
+
+<amplify-callout>
+Storage.copy can copy an object up to 5 GB in a single operation.
+</amplify-callout
+
+### Options
```

Comment:
Use H2

Reply:
or actually... maybe remove this line and instead say:

"To copy a file, you need to provide a source file `src` and a destination key `dest`"